src/test/jtx/permissioned_dex.h

```diff
@@ -0,0 +1,45 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2025 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#pragma once
+
+#include <test/jtx.h>
+#include <test/jtx/deposit.h>
+namespace ripple {
+namespace test {
+namespace jtx {
+
+class PermissionedDEX
```

Comment:
Most of the helper functions here just create/update transaction's JSON. This one is highly specialized  and don't belong here, better implement it in cpp file with the tests that used it. Hardcoded accounts and transactions will make explicit and implicit conflicts if someone wants to use this object in other tests.

Reply:
This class will be used across multiple test files. It's intentional to hardcode these accounts and transactions, otherwise, there will be a lot of redundant initialization that takes a lot space for every unit test. This object is not meant to be used with other kind of tests which will conflict with it.

Reply:
Ok, then at least please rename the accounts to not commonly used names in the tests, like add prefix to them or something
```
 gw("permdex-gateway")
, alice("permdex-alice")
 , bob("permdex-bob")
 ...
```

Reply:
2c1fe55a887e4199e8b19b050fa6c7c89e95d130